src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -1157,11 +1100,21 @@ def replace_hostname(self, endpoint, hostname_to_ip):
     def setup_multisite_replication(self, realm_name: str, zonegroup_name: str,
                                     zonegroup_endpoints: str, zone_name: str,
                                     zone_endpoints: str, username: str,
-                                    cluster_fsid: Optional[str] = None):
+                                    cluster_fsid: Optional[str] = None,
+                                    replication_zone_name: Optional[str] = None,
+                                    cluster_details: Optional[str] = None):
 
         # Set up multisite replication for Ceph RGW.
         logger.info("Starting multisite replication setup")
+        logger.info("cluster_details: %s", cluster_details)
+        if cluster_details:
+            cluster_details_dict = json.loads(cluster_details)
+            logger.info("cluster_details_dict: %s", cluster_details_dict)
```

Comment:
we are registering a lot of critical sensitive details in the log which is a security risk when you log these stuffs. rather than dumping the entire dict of items in the log, just focus on the items you want or the user care more like the cluster name, the gateway name, the user name and not the entire dict where it contains things like cluster token in some cases and rgw tokens in other.

Reply:
Were just added for debugging purposes, forgot to remove, removed now